modules/cli/pages/index.adoc

```diff
@@ -54,9 +56,146 @@ $ mv skupper $HOME/bin
+
+.Procedure
+
+. Determine the direction of the link. If both clusters are publicly addressable, then the direction is not significant. If one of the clusters is addressable from the other cluster, perform step 2 below on the addressable cluster.
+
+. Generate a token on the cluster that you want to link to:
++
+--
+----
+$ skupper token create <filename> 
+----
+
+where `<filename>` is the name of a YAML file that is saved on your local filesystem.
+
+This file contains a key and the location of the site that created it.
+
+NOTE: Access to this file provides access to the {service-network}. Protect it appropriately.
+--
+
+. Use a token to join the {service-network}:
++
+--
+----
+$ skupper link create <filename> [-name <link-name>]
+----
+where where `<filename>` is the name of a YAML file generated from the `skupper token create` command and `<link-name>` is the name of the link.
+--
+
+. If you want to delete a link:
++
+----
+$ skupper link delete <link-name>
+----
+where `<link-name>` is the name of the link specified during creation.
+
+
+// Type: procedure
+[id="exposing-services"] 
+== Exposing services on the {service-network}
+
+After creating a {service-network}, services can communicate across that network.
+This section describes how services can be enabled for a {service-network}.
+
+.Procedure
+
+. Create a deployment, some pods, or a service in one of your sites, for example to create the backend service for the link:{getting-started-link}[tutorial]:
++
+----
+$ kubectl create deployment hello-world-backend --image quay.io/skupper/hello-world-backend
+----
++
+This step is not {skupper-name}-specific, that is, this process is unchanged from standard processes for your cluster.
+
+
+. Expose the service on the  {service-network} using the `expose` option:
++
+--
+----
+$ skupper expose [deployment <name>|pods <selector>|service <name>]
+----
+
+where `<name>` is the name of your deployment or service and `<selector>` is a pod selector.
+
+Options for this command include:
+
+`--port <port-number>`:: Specify the port number that this service is available on the {service-network}.
+
+`--target-port <port-number>`:: Specify the port number of pods that you want to expose.
+--
+
+
+// Type: reference
+[id="cli-global-options"] 
+== CLI options for working with different clusters
+
+By default, all `skupper` commands apply to the cluster you are logged into and the current namespace.
+The following `skupper` options allow you to override that behavior and apply to all commands:
+
+`--namespace <namespace-name>`:: Apply command to `<namespace-name`. For example, if you are currently working on `frontend` namespace and want to initialize a site in the `backend` namespace:
```

Comment:
... Apply command to `<namespace-name` (guess there is an extra "<" in this sentence.